packages/client/hmi-client/src/components/widgets/tera-common-modal-dialogs.vue

```diff
@@ -96,6 +96,9 @@ watch(
 	async () => {
 		if (isCopyDialogVisible.value) {
 			project.value = await useProjects().activeProject.value;
+			if (project.value === null) {
+				project.value = menuProject.value;
+			}
 			const name = project.value.name ?? null;
 			projectName.value = name ? `Copy of ${name}` : 'Enter new project name';
 		}
```

Comment:
We have `getActiveProjectName()` available.

```suggestion
		if (isCopyDialogVisible.value) {
			const name = await useProjects().getActiveProjectName();
			projectName.value = isEmpty(name) ? 'Enter new project name' : `Copy of ${name}`;
		}
```